vale/styles/spectrocloud/file-type.yml

```diff
@@ -0,0 +1,16 @@
+extends: substitution
+message: "Incorrect file type reference. Use '%s'."
+level: error
+ignorecase: false
+swap:
+  csv file: CSV file
+  json file: JSON file
+  png file: PNG file
+  svg file: SVG file
+  TAR file: tar file
+  tf file: Terraform file
+  txt file: text file
+  yaml file: YAML file
+  ZIP file: zip file
+  ZST file: zst file
+  TGZ file: tgz file
```

Comment:
These seem bizarre to me when they're not in ALL CAPS.

Reply:
I agree